packages/playground/tsconfig.json

```
@@ -19,18 +28,45 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "types": ["vite/client", "jest", "node"]
+    "types": [
+      "vite/client",
+      "jest",
+      "node"
+    ]
   },
   "references": [
-    { "path": "../antd" },
-    { "path": "../chakra-ui" },
-    { "path": "../core" },
-    { "path": "../fluentui-rc" },
-    { "path": "../mui" },
-    { "path": "../react-bootstrap"},
-    { "path": "../semantic-ui" },
-    { "path": "../shadcn" },
-    { "path": "../utils" },
-    { "path": "../validator-ajv8" }
+    {
+      "path": "../antd"
+    },
+    {
+      "path": "../chakra-ui"
+    },
+    {
+      "path": "../core"
+    },
+    {
+      "path": "../daisyui"
+    },
+    {
+      "path": "../fluentui-rc"
+    },
+    {
+      "path": "../mui"
+    },
+    {
+      "path": "../react-bootstrap"
+    },
+    {
+      "path": "../semantic-ui"
+    },
+    {
+      "path": "../shadcn"
+    },
+    {
+      "path": "../utils"
+    },
+    {
+      "path": "../validator-ajv8"
+    }
   ]
-}
+}
```

Adding blank line at end to remove the red circled dash
```suggestion
}

```